test/unit/api/api.test.js

```diff
@@ -43,15 +43,19 @@ const createApp = (locals) => {
 
 describe('API', () => {
   describe('Authentication', () => {
-    const app = createApp();
+    let app;
 
-    it('should return 404 if no token is provided', () => supertest(app)
-      .get('/')
-      .set('Authorization', 'Bearer xxx')
-      .expect(404)
-      .then(response => {
-        expect(response.body).toMatchSnapshot();
-      }));
+    beforeAll(() => {
+      app = createApp();
+    });
+
+    it('should return 404 if no token is provided', () =>
+      supertest(app)
```

Comment:
This test passed due to the fact that the old url it was testing was `/` and that route wasn't ever set up. Now setting to `/` and removed the `Authorization` header to test what the test meant to test.


....Test.